src/codegen/metadata/script_info.py

```diff
@@ -0,0 +1,8 @@
+script_info = {
```

Comment:
consider having the whole moduleinfo instead of having only the necessary key values